src/com/pentalog/pentastagiu/homework2/problem2/HomeworkApp.java

```diff
@@ -0,0 +1,31 @@
+package com.pentalog.pentastagiu.homework2.problem2;
```

Comment:
tinand cont ca aici testezi toate cele 3 probleme, ar trebui sa muti clasa asta un nivel mai sus, direct in homework2